src/utilities.js

```diff
@@ -1,36 +1,14 @@
 import R from 'ramda';
 import { NpmConfig, httpsGetPromise } from './helpers';
 
-const addVersionProp = (val, key, obj) => obj[key] = { 'version': removeCaret(val) };
 const removeCaret = R.replace(/\^/, '');
+const addVersionProp = (v, k, o) => o[k] = { 'version': removeCaret(v) };
 const isNotPrivate = R.compose(R.not, R.prop('private'));
-const filterPrivate = R.filter(isNotPrivate);
-
-const deeplyMerge = (obj1, obj2) => {
-  return R.keys(obj1).reduce((result, key) => {
-    result[key] = R.merge(obj1[key], obj2[key]);
-    return result;
-  }, {});
-};
-
-export const pickDownloads = (obj) => {
-  return R.keys(obj).reduce((result, key) => {
-    result[key] = R.pick(['downloads'], obj[key]);
-    return result;
-  }, {});
-};
-
-export const addNode = (obj) => {
-  obj.node = {
-    name        : 'Node.js',
-    url         : 'https://nodejs.org',
-    version     : process.versions.node,
-    description : 'A JavaScript runtime ✨🐢🚀✨',
-    downloads   : 10000000 // A fake number since Node isn't downloaded on npm
-  };
-  return obj;
-};
+const deeplyMerge = (obj1, obj2) => R.mapObjIndexed((v, k, o) => R.merge(obj1[k], obj2[k]), obj1);
```

Comment:
We should also consider exporting these to make them easier to test. Presently we're only testing parts of the module's API. It would give me some piece of mind to test their innards, as well.

I'm thinking we throw in something like:

```
export default utilities = {
  removeCaret: removeCaret,
  // ...,
  // ...,
};
```

or we throw an export in front of each of the functions.

Thoughts?